exporter/awsemfexporter/util_test.go

```diff
@@ -182,6 +182,7 @@ func TestGetLogInfo(t *testing.T) {
 			Labels: map[string]string{
 				"aws.ecs.cluster.name": "test-cluster-name",
 				"aws.ecs.task.id":      "test-task-id",
+				"NodeName":             "ip-192-168-58-245.ec2.internal",
```

Comment:
should the we also test `k8s.node.name` as key?

Reply:
Sure.